examples/guestbook-go/redis-master-service.json

```diff
@@ -12,7 +12,8 @@
       "ports": [
         {
           "port":6379,
-          "targetPort":"redis-server"
+          "targetPort":"redis-server",
+          "nodePort":0
```

Comment:
Yes I see that it was changed recently: https://github.com/kubernetes/kubernetes/pull/12831
http://kubernetes.io/third_party/swagger-ui has not been updated yet, but e2e should be using https://github.com/kubernetes/kubernetes/tree/master/api/swagger-spec which is updated.